### DISCUSSION TITLE
Add Support for filter-display="menu" in TreeTable

### DISCUSSION THREAD

**Discussion by arjunarmino** [primefaces/community#2989] [PrimeVue]
I’d like to request adding support for the filter-display="menu" option in the TreeTable component. Currently, DataTable supports this feature, allowing filters to be shown in a separate menu, which is especially useful for tables with multiple columns and complex filtering requirements.

In the TreeTable component, however, filtering options are limited to inline filters within the column headers. It would be great to have the filter-display="menu" functionality in TreeTable as well, as this would:

Enhance UX by decluttering the header area.
Allow more advanced and flexible filtering options, especially for datasets with nested structures.
Create a consistent filtering experience between DataTable and TreeTable.

**Comment by nekorrrr** [comment]
I think it's a great idea and it would be great to add this feature to Treetable!

**Comment by anchan42** [comment]
Look forward for this feature too
